src/DetailsView/Styles/detailsview.scss

```diff
@@ -867,7 +867,7 @@ div.insights-file-issue-details-dialog-container {
                 min-height: 0;
                 .target-page-changed,
                 .static-content-details-view {
-                    margin: 24px 24px 0px 24px;
+                    margin: $fastPassRightPanelMarginTop $fastPassRightPanelMarginLeft auto $fastPassRightPanelMarginLeft;
```

Comment:
Was the second term in this value meant to be $fastPassRightPanelMargin**Right**?

Reply:
Yes! Updated!